pom.xml

```diff
@@ -963,18 +963,6 @@
 			</extension>
 		</extensions>
 	</build>
-	<repositories>
-		<repository>
-			<releases>
-				<enabled>false</enabled>
-			</releases>
-			<snapshots>
-				<enabled>true</enabled>
-			</snapshots>
-			<id>oss-sonatype-snapshots</id>
-			<url>https://oss.sonatype.org/content/repositories/snapshots</url>
-		</repository>
-	</repositories>
```

Comment:
Wait, this should not be removed.

Reply:
Ahh. Forgot. I remember now that I started working on this branch once when master was broken. Will take a good look and see if there is anything else I forgot.